### ISSUE TITLE
Active Effects

### ISSUE THREAD

**Issue by akrigline** [zeel01/MonsterBlocks#130]
The core system has started ramping up its usage of active effects and modules like DFred's Convenient Effects have gained in popularity pretty dramatically lately.

It would be excellent to have a display of effects with the basic controls to remove/deactivate them on the sheet.

**Comment by zeel01** [comment]
Yes, this is something I'll include when I revisit the feature sorting and sections system. I need to separate Bonus Actions out as well, and probably make some changes to how spell casters are handled too.
